commands/team_users_test.go

```diff
@@ -66,10 +66,10 @@ func (s *MmctlUnitTestSuite) TestTeamUsersArchiveCmd() {
 			Times(1)
 
 		err := teamUsersRemoveCmdF(s.client, &cobra.Command{}, []string{teamArg, mockUser.Id})
-		s.Require().Nil(err)
+		s.Require().NotNil(err)
```

Comment:
You can use `s.Require().Error(err)` here. That makes the error messages easier to understand.